client/verta/verta/_registry/modelversion.py

```diff
@@ -511,11 +511,15 @@ def _upload_artifact(self, key, file_handle, artifact_type, part_size=64*(10**6)
 
                 # upload part
                 #     Retry connection errors, to make large multipart uploads more robust.
-                for _ in range(3):
+                for i in range(3):
```

Comment:
Ooh, let's make the number of tries a variable so we can avoid hard-coding this `3` and the later `2`.
Also check that `MAX_TRIES` isn't already being used.
```suggestion
                MAX_TRIES = 3
                for i in range(MAX_TRIES):
```

Reply:
You got it!